pandas/core/frame.py

```diff
@@ -2121,9 +2121,11 @@ def info(self, verbose=None, buf=None, max_cols=None, memory_usage=None,
         <class 'pandas.core.frame.DataFrame'>
         RangeIndex: 5 entries, 0 to 4
         Data columns (total 3 columns):
-        int_col      5 non-null int64
-        text_col     5 non-null object
-        float_col    5 non-null float64
+         #.  Column       Non-Null Count
```

Comment:
maybe want to call this
``Non-Null Count & Dtype``? 

will need to adjust this header if ``null_counts=False`` is passed to ``.info()`` (add a tests as well)

Reply:
for `df.info(null_counts=False)` header will have `dtype`, or would you prefer `Dtype` -- is that fine?

```
In [38]: df.info(null_counts=False)
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 6 entries, 0 to 5
Data columns (total 2 columns):
 #.  Column    dtype
---  ------    -----
 0   foo1      object
 1   foo2      float64
dtypes: float64(1), object(1)
memory usage: 168.0+ bytes
```